assets/anim.scss

```diff
@@ -1,20 +1,235 @@
-@import "common";
-
-// infinite loop animation option
-@mixin pngAnimate($animation, $duration, $timingFunction, $iterationCount) {
-  animation: $animation $duration $timingFunction $iterationCount;
-  &:hover {
-    animation: $animation $duration $timingFunction infinite;
-  }
-  &-animate-forever {
-    animation: $animation $duration $timingFunction infinite;
-  }
-}
+@import 'common';
+
+$emote-prefix: '.chat-emote-';
+$css-animated-emote-animations: ();
+$context-css-animated-emote-mixin: false;
+
+// Mixin for defining an emote with a css animation.
+// Should be used in combination with emote-animation mixin.
+@mixin css-animated-emote($name, $can-animate-forever: false) {
```

Comment:
I wonder if `$can-animate-forever` should default to `true`?

Reply:
Currently DJPepo is the only css animated emote that uses it.
I would stick to what is the most common configuration.

Reply:
Before this patch, all animated PNG emotes had the `&-animate-forever` selector. Now only DJPepo will?

Reply:
No, the animated PNGs(defined by sprite-animation mixin) all animate per default.
PJPepo uses css animation, not a sprite. 
So it uses the same mixin as stuff like "OverRustle" and "DANKMEMES".

I would split those categories up in to 2 files, honestly.
And put the remaining animations with the classes that use them.
But I left that for another day.

Reply:
Ah, missed that mixin.